mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:property name="M_PI" value="3.1415926535897931" />
+
+    <xacro:macro name="range_beam_model" params="name parent *origin">
+      <joint name="${name}_joint" type="fixed">
+        <xacro:insert_block name="origin" />
+        <parent link="${parent}"/>
+        <child link="${name}_frame"/>
```

Comment:
remove extra `_frame`

Reply:
I removed the lidarlite.urdf.xacro file 
This is a file I added at first and than didn't use it